apps/src/templates/sectionsRefresh/SectionsSetUpContainer.jsx

```diff
@@ -83,7 +84,8 @@ const saveSection = (e, section) => {
 };
 
 // TO DO: Add a prop to indicate if this is a new section or an existing section
-export default function SectionsSetUpContainer() {
+export default function SectionsSetUpContainer(props) {
+  const {isNewSection} = props;
```

Comment:
nit: with functional components you can destruct props in the parameter passed to the function, ie
```
export default function SectionsSetUpContainer({isNewSection}) {
```